Contact dashboard takes long too load

<!--
Please read the Contributing guidelines (https://github.com/hzi-braunschweig/SORMAS-Project/blob/development/docs/CONTRIBUTING.md) before submitting an issue. You don't have to remove this comment or any other comment from this issue as they will automatically be hidden.
-->
### Bug Description
<!-- Mandatory -->
After a long loading time the dashboard opens with an error message. Nevertheless data are displayed.
The error was observed only with a Contact Supervisor (ContSup) in the Region "Voreingestellte Bundesländer"

### Steps to Reproduce
<!-- Optional; please add more steps if necessary -->
1. Log in a ContSup
2. Click on Dashboard


### Expected Behavior
<!-- Optional -->

### Screenshots
<!-- Optional -->
![image](https://user-images.githubusercontent.com/101708230/190607048-19854eaa-1d0b-41d4-82cc-5a8f4de44fc0.png)


### System Details
<!-- Mandatory; you only have to specify the Server URL if the error appeared on a publicly available test server -->
* Device: Laptop
* SORMAS version:   1.76.0  - 5201c4d
* Android version/Browser: Chrome
* Server URL:  https://test-de.sormas.netzlink.com/ and https://test-auto.sormas.netzlink.com/
* User Role: ContSup

### Additional Information
<!-- Optional -->

~~Logfile: [Log.txt](https://github.com/hzi-braunschweig/SORMAS-Project/files/9583637/Log.txt)~~ not performance related

2022-09-15 14:41:10	... 106 common frames omitted2022-09-15 14:41:10	at com.sun.corba.ee.impl.presentation.rmi.StubInvocationHandlerImpl.privateInvoke(StubInvocationHandlerImpl.java:188)2022-09-15 14:41:10	at com.sun.corba.ee.impl.protocol.ClientDelegateImpl.invoke(ClientDelegateImpl.java:258)2022-09-15 14:41:10	at com.sun.corba.ee.impl.protocol.SharedCDRClientRequestDispatcherImpl.marshalingComplete(SharedCDRClientRequestDispatcherImpl.java:166)2022-09-15 14:41:10	at com.sun.corba.ee.impl.protocol.ClientRequestDispatcherImpl.processResponse(ClientRequestDispatcherImpl.java:506)2022-09-15 14:41:10	at com.sun.corba.ee.impl.protocol.MessageMediatorImpl.getSystemExceptionReply(MessageMediatorImpl.java:610)2022-09-15 14:41:10	at com.sun.corba.ee.impl.protocol.giopmsgheaders.ReplyMessage_1_2.getSystemException(ReplyMessage_1_2.java:131)2022-09-15 14:41:10	at com.sun.corba.ee.impl.protocol.giopmsgheaders.MessageBase.getSystemException(MessageBase.java:813)2022-09-15 14:41:10	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)2022-09-15 14:41:10----------END server-side stack trace----------2022-09-15 14:41:102022-09-15 14:41:10	... 120 more2022-09-15 14:41:10	at com.sun.ejb.containers.EJBObjectInvocationHandler.invoke(EJBObjectInvocationHandler.java:205)2022-09-15 14:41:10	at com.sun.ejb.containers.BaseContainer.intercept(BaseContainer.java:4795)2022-09-15 14:41:10	at com.sun.ejb.containers.BaseContainer.__intercept(BaseContainer.java:4807)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.InterceptorManager.intercept(InterceptorManager.java:375)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)2022-09-15 14:41:10	at java.base/java.lang.reflect.Method.invoke(Method.java:566)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)2022-09-15 14:41:10	at jdk.internal.reflect.GeneratedMethodAccessor241.invoke(Unknown Source)2022-09-15 14:41:10	at org.jboss.weld.module.ejb.SessionBeanInterceptor.aroundInvoke(SessionBeanInterceptor.java:52)2022-09-15 14:41:10	at org.jboss.weld.module.ejb.AbstractEJBRequestScopeActivationInterceptor.aroundInvoke(AbstractEJBRequestScopeActivationInterceptor.java:81)2022-09-15 14:41:10	at com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)2022-09-15 14:41:10	at java.base/java.lang.reflect.Method.invoke(Method.java:566)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)2022-09-15 14:41:10	at jdk.internal.reflect.GeneratedMethodAccessor251.invoke(Unknown Source)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.SystemInterceptorProxy.aroundInvoke(SystemInterceptorProxy.java:140)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.SystemInterceptorProxy.doCall(SystemInterceptorProxy.java:163)2022-09-15 14:41:10	at com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)2022-09-15 14:41:10	at java.base/java.lang.reflect.Method.invoke(Method.java:566)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)2022-09-15 14:41:10	at jdk.internal.reflect.GeneratedMethodAccessor252.invoke(Unknown Source)2022-09-15 14:41:10	at de.symeda.sormas.backend.audit.AuditLoggerInterceptor.logAudit(AuditLoggerInterceptor.java:78)2022-09-15 14:41:10	at com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)2022-09-15 14:41:10	at java.base/java.lang.reflect.Method.invoke(Method.java:566)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)2022-09-15 14:41:10	at jdk.internal.reflect.GeneratedMethodAccessor253.invoke(Unknown Source)2022-09-15 14:41:10	at de.symeda.sormas.backend.util.PerformanceLoggingInterceptor.logInvokeDuration(PerformanceLoggingInterceptor.java:37)2022-09-15 14:41:10	at com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)2022-09-15 14:41:10	at java.base/java.lang.reflect.Method.invoke(Method.java:566)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)2022-09-15 14:41:10	at jdk.internal.reflect.GeneratedMethodAccessor254.invoke(Unknown Source)2022-09-15 14:41:10	at de.symeda.sormas.backend.util.RightsAllowedInterceptor.checkRightsAllowed(RightsAllowedInterceptor.java:111)2022-09-15 14:41:10	at com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)2022-09-15 14:41:10	at java.base/java.lang.reflect.Method.invoke(Method.java:566)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)2022-09-15 14:41:10	at jdk.internal.reflect.GeneratedMethodAccessor255.invoke(Unknown Source)2022-09-15 14:41:10	at de.symeda.sormas.backend.util.ValidationConstraintViolationInterceptor.handleValidationConstraintViolation(ValidationConstraintViolationInterceptor.java:35)2022-09-15 14:41:10	at com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:834)2022-09-15 14:41:10	at com.sun.ejb.EjbInvocation.invokeBeanMethod(EjbInvocation.java:665)2022-09-15 14:41:10	at com.sun.ejb.containers.BaseContainer.invokeBeanMethod(BaseContainer.java:4835)2022-09-15 14:41:10	at org.glassfish.ejb.security.application.EJBSecurityManager.invoke(EJBSecurityManager.java:408)2022-09-15 14:41:10	at org.glassfish.ejb.security.application.EJBSecurityManager.runMethod(EJBSecurityManager.java:588)2022-09-15 14:41:10	at java.base/java.lang.reflect.Method.invoke(Method.java:566)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)2022-09-15 14:41:10	at de.symeda.sormas.backend.contact.ContactFacadeEjb.getContactCountsByCasesForDashboard(ContactFacadeEjb.java:1295)2022-09-15 14:41:10	at com.sun.enterprise.container.common.impl.EntityManagerWrapper.createQuery(EntityManagerWrapper.java:498)2022-09-15 14:41:10	at org.hibernate.internal.AbstractSessionImpl.createQuery(AbstractSessionImpl.java:23)2022-09-15 14:41:10	at org.hibernate.internal.AbstractSharedSessionContract.createQuery(AbstractSharedSessionContract.java:774)2022-09-15 14:41:10	at org.hibernate.query.criteria.internal.compile.CriteriaCompiler.compile(CriteriaCompiler.java:165)2022-09-15 14:41:10	at org.hibernate.query.criteria.internal.CriteriaQueryImpl$1.buildCompiledQuery(CriteriaQueryImpl.java:364)2022-09-15 14:41:10	at org.hibernate.query.criteria.internal.compile.CriteriaCompiler$1$1.bind(CriteriaCompiler.java:135)2022-09-15 14:41:10	at org.hibernate.query.internal.AbstractProducedQuery.setParameter(AbstractProducedQuery.java:122)2022-09-15 14:41:10	at org.hibernate.query.internal.AbstractProducedQuery.setParameter(AbstractProducedQuery.java:501)2022-09-15 14:41:10	at org.hibernate.query.internal.QueryParameterBindingImpl.setBindValue(QueryParameterBindingImpl.java:55)2022-09-15 14:41:10	at org.hibernate.query.internal.QueryParameterBindingImpl.validate(QueryParameterBindingImpl.java:90)2022-09-15 14:41:10	at org.hibernate.query.spi.QueryParameterBindingValidator.validate(QueryParameterBindingValidator.java:27)2022-09-15 14:41:10	at org.hibernate.query.spi.QueryParameterBindingValidator.validate(QueryParameterBindingValidator.java:54)2022-09-15 14:41:10Caused by: java.lang.IllegalArgumentException: Parameter value [8896888] did not match expected type [de.symeda.sormas.backend.caze.Case (n/a)]2022-09-15 14:41:10	... 122 more2022-09-15 14:41:10	at com.sun.ejb.containers.BaseContainer.postInvoke(BaseContainer.java:2134)2022-09-15 14:41:10	at com.sun.ejb.containers.BaseContainer.postInvokeTx(BaseContainer.java:4601)2022-09-15 14:41:10	at com.sun.ejb.containers.EJBContainerTransactionManager.postInvokeTx(EJBContainerTransactionManager.java:485)2022-09-15 14:41:10	at com.sun.ejb.containers.EJBContainerTransactionManager.checkExceptionClientTx(EJBContainerTransactionManager.java:615)2022-09-15 14:41:10Caused by: javax.ejb.TransactionRolledbackLocalException: Exception thrown from bean: java.lang.IllegalArgumentException: Parameter value [8896888] did not match expected type [de.symeda.sormas.backend.caze.Case (n/a)]2022-09-15 14:41:10	at java.base/java.lang.Thread.run(Thread.java:829)2022-09-15 14:41:10	at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.run(AbstractThreadPool.java:549)2022-09-15 14:41:10	at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.doWork(AbstractThreadPool.java:569)2022-09-15 14:41:10	at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy$WorkerThreadRunnable.run(WorkerThreadIOStrategy.java:114)2022-09-15 14:41:10	at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.access$100(WorkerThreadIOStrategy.java:33)2022-09-15 14:41:10	at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.run0(WorkerThreadIOStrategy.java:94)2022-09-15 14:41:10	at org.glassfish.grizzly.strategies.AbstractIOStrategy.fireIOEvent(AbstractIOStrategy.java:89)2022-09-15 14:41:10	at org.glassfish.grizzly.nio.transport.TCPNIOTransport.fireIOEvent(TCPNIOTransport.java:524)2022-09-15 14:41:10	at org.glassfish.grizzly.ProcessorExecutor.execute(ProcessorExecutor.java:53)2022-09-15 14:41:10	at org.glassfish.grizzly.filterchain.DefaultFilterChain.process(DefaultFilterChain.java:88)2022-09-15 14:41:10	at org.glassfish.grizzly.filterchain.DefaultFilterChain.execute(DefaultFilterChain.java:109)2022-09-15 14:41:10	at org.glassfish.grizzly.filterchain.DefaultFilterChain.executeChainPart(DefaultFilterChain.java:177)2022-09-15 14:41:10	at org.glassfish.grizzly.filterchain.DefaultFilterChain.executeFilter(DefaultFilterChain.java:260)2022-09-15 14:41:10	at org.glassfish.grizzly.filterchain.ExecutorResolver$9.execute(ExecutorResolver.java:95)2022-09-15 14:41:10	at org.glassfish.grizzly.http.server.HttpServerFilter.handleRead(HttpServerFilter.java:218)2022-09-15 14:41:10	at org.glassfish.grizzly.http.server.HttpHandler.doHandle(HttpHandler.java:156)2022-09-15 14:41:10	at org.glassfish.grizzly.http.server.HttpHandler.runService(HttpHandler.java:182)2022-09-15 14:41:10	at com.sun.enterprise.v3.services.impl.ContainerMapper.service(ContainerMapper.java:217)2022-09-15 14:41:10	at com.sun.enterprise.v3.services.impl.ContainerMapper$HttpHandlerCallable.call(ContainerMapper.java:520)2022-09-15 14:41:10	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:238)2022-09-15 14:41:10	at org.apache.catalina.connector.CoyoteAdapter.doService(CoyoteAdapter.java:371)2022-09-15 14:41:10	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:158)2022-09-15 14:41:10	at com.sun.enterprise.web.WebPipeline.invoke(WebPipeline.java:97)2022-09-15 14:41:10	at org.apache.catalina.core.StandardPipeline.doChainInvoke(StandardPipeline.java:581)2022-09-15 14:41:10	at org.apache.catalina.core.StandardPipeline.doInvoke(StandardPipeline.java:757)2022-09-15 14:41:10	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:160)2022-09-15 14:41:10	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:257)2022-09-15 14:41:10	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:211)2022-09-15 14:41:10	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:253)2022-09-15 14:41:10	at de.symeda.sormas.ui.Slf4jMdcFilter.doFilter(Slf4jMdcFilter.java:29)2022-09-15 14:41:10	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:211)2022-09-15 14:41:10	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:253)2022-09-15 14:41:10	at de.symeda.sormas.ui.SessionFilter.doFilter(SessionFilter.java:66)2022-09-15 14:41:10	at de.symeda.sormas.ui.__EJB31_Generated__SessionFilterBean__Intf____Bean__.doFilter(Unknown Source)2022-09-15 14:41:10	at com.sun.proxy.$Proxy893.doFilter(Unknown Source)2022-09-15 14:41:10	at com.sun.ejb.containers.EJBLocalObjectInvocationHandlerDelegate.invoke(EJBLocalObjectInvocationHandlerDelegate.java:90)2022-09-15 14:41:10	at com.sun.ejb.containers.EJBLocalObjectInvocationHandler.invoke(EJBLocalObjectInvocationHandler.java:212)2022-09-15 14:41:10	at com.sun.ejb.containers.BaseContainer.intercept(BaseContainer.java:4795)2022-09-15 14:41:10	at com.sun.ejb.containers.BaseContainer.__intercept(BaseContainer.java:4807)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.InterceptorManager.intercept(InterceptorManager.java:375)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)2022-09-15 14:41:10	at java.base/java.lang.reflect.Method.invoke(Method.java:566)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)2022-09-15 14:41:10	at jdk.internal.reflect.GeneratedMethodAccessor241.invoke(Unknown Source)2022-09-15 14:41:10	at org.jboss.weld.module.ejb.SessionBeanInterceptor.aroundInvoke(SessionBeanInterceptor.java:52)2022-09-15 14:41:10	at org.jboss.weld.module.ejb.AbstractEJBRequestScopeActivationInterceptor.aroundInvoke(AbstractEJBRequestScopeActivationInterceptor.java:72)2022-09-15 14:41:10	at com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)2022-09-15 14:41:10	at java.base/java.lang.reflect.Method.invoke(Method.java:566)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)2022-09-15 14:41:10	at jdk.internal.reflect.GeneratedMethodAccessor251.invoke(Unknown Source)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.SystemInterceptorProxy.aroundInvoke(SystemInterceptorProxy.java:140)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.SystemInterceptorProxy.doCall(SystemInterceptorProxy.java:163)2022-09-15 14:41:10	at com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:834)2022-09-15 14:41:10	at com.sun.ejb.EjbInvocation.invokeBeanMethod(EjbInvocation.java:665)2022-09-15 14:41:10	at com.sun.ejb.containers.BaseContainer.invokeBeanMethod(BaseContainer.java:4835)2022-09-15 14:41:10	at org.glassfish.ejb.security.application.EJBSecurityManager.invoke(EJBSecurityManager.java:408)2022-09-15 14:41:10	at org.glassfish.ejb.security.application.EJBSecurityManager.runMethod(EJBSecurityManager.java:588)2022-09-15 14:41:10	at java.base/java.lang.reflect.Method.invoke(Method.java:566)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)2022-09-15 14:41:10	at jdk.internal.reflect.GeneratedMethodAccessor467.invoke(Unknown Source)2022-09-15 14:41:10	at de.symeda.sormas.ui.SessionFilterBean.doFilter(SessionFilterBean.java:37)2022-09-15 14:41:10	at de.symeda.sormas.ui.SessionFilter.lambda$doFilter$2(SessionFilter.java:73)2022-09-15 14:41:10	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:211)2022-09-15 14:41:10	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:331)2022-09-15 14:41:10	at org.apache.catalina.core.StandardWrapper.service(StandardWrapper.java:1637)2022-09-15 14:41:10	at javax.servlet.http.HttpServlet.service(HttpServlet.java:750)2022-09-15 14:41:10	at com.vaadin.server.VaadinServlet.service(VaadinServlet.java:464)2022-09-15 14:41:10	at com.vaadin.server.VaadinService.handleRequest(VaadinService.java:1637)2022-09-15 14:41:10	at com.vaadin.server.SynchronizedRequestHandler.handleRequest(SynchronizedRequestHandler.java:40)2022-09-15 14:41:10	at com.vaadin.server.communication.UidlRequestHandler.synchronizedHandleRequest(UidlRequestHandler.java:91)2022-09-15 14:41:10	at com.vaadin.server.communication.ServerRpcHandler.handleRpc(ServerRpcHandler.java:275)2022-09-15 14:41:10	at com.vaadin.server.communication.ServerRpcHandler.handleInvocations(ServerRpcHandler.java:407)2022-09-15 14:41:10	at com.vaadin.server.communication.ServerRpcHandler.handleInvocation(ServerRpcHandler.java:442)2022-09-15 14:41:10	at com.vaadin.server.ServerRpcManager.applyInvocation(ServerRpcManager.java:116)2022-09-15 14:41:10	at com.vaadin.server.ServerRpcManager.applyInvocation(ServerRpcManager.java:155)2022-09-15 14:41:10	at java.base/java.lang.reflect.Method.invoke(Method.java:566)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)2022-09-15 14:41:10	at jdk.internal.reflect.GeneratedMethodAccessor787.invoke(Unknown Source)2022-09-15 14:41:10	at com.vaadin.ui.Button$1.click(Button.java:57)2022-09-15 14:41:10	at com.vaadin.ui.Button.fireClick(Button.java:384)2022-09-15 14:41:10	at com.vaadin.server.AbstractClientConnector.fireEvent(AbstractClientConnector.java:1190)2022-09-15 14:41:10	at com.vaadin.event.EventRouter.fireEvent(EventRouter.java:363)2022-09-15 14:41:10	at com.vaadin.event.EventRouter.fireEvent(EventRouter.java:399)2022-09-15 14:41:10	at com.vaadin.event.ListenerMethod.receiveEvent(ListenerMethod.java:706)2022-09-15 14:41:10	at java.base/java.lang.reflect.Method.invoke(Method.java:566)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)2022-09-15 14:41:10	at jdk.internal.reflect.GeneratedMethodAccessor769.invoke(Unknown Source)2022-09-15 14:41:10	at de.symeda.sormas.ui.Menu.lambda$createViewButton$f7d2215a$1(Menu.java:196)2022-09-15 14:41:10	at com.vaadin.navigator.Navigator.navigateTo(Navigator.java:678)2022-09-15 14:41:10	at com.vaadin.navigator.Navigator.navigateTo(Navigator.java:701)2022-09-15 14:41:10	at com.vaadin.navigator.Navigator.runAfterLeaveConfirmation(Navigator.java:730)2022-09-15 14:41:10	at com.vaadin.navigator.View.beforeLeave(View.java:79)2022-09-15 14:41:10	at com.vaadin.navigator.ViewBeforeLeaveEvent.navigate(ViewBeforeLeaveEvent.java:54)2022-09-15 14:41:10	at com.vaadin.navigator.Navigator.lambda$navigateTo$9a874efd$1(Navigator.java:702)2022-09-15 14:41:10	at com.vaadin.navigator.Navigator.performNavigateTo(Navigator.java:778)2022-09-15 14:41:10	at de.symeda.sormas.ui.dashboard.contacts.ContactsDashboardView.enter(ContactsDashboardView.java:144)2022-09-15 14:41:10	at de.symeda.sormas.ui.dashboard.contacts.ContactsDashboardView.refreshDashboard(ContactsDashboardView.java:475)2022-09-15 14:41:10	at de.symeda.sormas.ui.dashboard.contacts.ContactsDashboardView.updateCaseCountsAndSourceCasesLabels(ContactsDashboardView.java:269)2022-09-15 14:41:10	at de.symeda.sormas.api.contact._ContactFacade_Wrapper.getContactCountsByCasesForDashboard(de/symeda/sormas/api/contact/_ContactFacade_Wrapper.java)2022-09-15 14:41:10	at de.symeda.sormas.api.contact.__ContactFacade_Remote_DynamicStub.getContactCountsByCasesForDashboard(de/symeda/sormas/api/contact/__ContactFacade_Remote_DynamicStub.java)2022-09-15 14:41:10	at com.sun.corba.ee.impl.presentation.rmi.codegen.CodegenStubBase.invoke(CodegenStubBase.java:226)2022-09-15 14:41:10	at com.sun.corba.ee.impl.presentation.rmi.StubInvocationHandlerImpl.invoke(StubInvocationHandlerImpl.java:140)2022-09-15 14:41:10	at com.sun.corba.ee.impl.presentation.rmi.StubInvocationHandlerImpl.privateInvoke(StubInvocationHandlerImpl.java:188)2022-09-15 14:41:10	at com.sun.corba.ee.impl.protocol.ClientDelegateImpl.invoke(ClientDelegateImpl.java:258)2022-09-15 14:41:10	at com.sun.corba.ee.impl.protocol.SharedCDRClientRequestDispatcherImpl.marshalingComplete(SharedCDRClientRequestDispatcherImpl.java:128)2022-09-15 14:41:10	at com.sun.corba.ee.impl.protocol.MessageMediatorImpl.handleRequestRequest(MessageMediatorImpl.java:1565)2022-09-15 14:41:10	at com.sun.corba.ee.impl.protocol.ServerRequestDispatcherImpl.dispatch(ServerRequestDispatcherImpl.java:199)2022-09-15 14:41:10	at com.sun.corba.ee.impl.protocol.ServerRequestDispatcherImpl.dispatchToServant(ServerRequestDispatcherImpl.java:528)2022-09-15 14:41:10	at com.sun.corba.ee.impl.presentation.rmi.ReflectiveTie._invoke(ReflectiveTie.java:173)2022-09-15 14:41:10	at com.sun.corba.ee.impl.presentation.rmi.ReflectiveTie.dispatchToMethod(ReflectiveTie.java:143)2022-09-15 14:41:10	at java.base/java.lang.reflect.Method.invoke(Method.java:566)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)2022-09-15 14:41:10	at com.sun.proxy.$Proxy682.getContactCountsByCasesForDashboard(Unknown Source)2022-09-15 14:41:10	at com.sun.ejb.containers.EJBObjectInvocationHandlerDelegate.invoke(EJBObjectInvocationHandlerDelegate.java:79)2022-09-15 14:41:10	at com.sun.ejb.containers.EJBObjectInvocationHandler.invoke(EJBObjectInvocationHandler.java:212)2022-09-15 14:41:10	at com.sun.ejb.containers.BaseContainer.postInvoke(BaseContainer.java:2104)2022-09-15 14:41:10	at com.sun.ejb.containers.BaseContainer.postInvoke(BaseContainer.java:2173)2022-09-15 14:41:10	at com.sun.ejb.containers.BaseContainer.mapRemoteException(BaseContainer.java:2330)2022-09-15 14:41:10	at com.sun.ejb.containers.BaseContainer.mapRemoteException(BaseContainer.java:2357)2022-09-15 14:41:10	at org.glassfish.enterprise.iiop.impl.POAProtocolMgr.mapException(POAProtocolMgr.java:356)2022-09-15 14:41:10org.omg.CORBA.TRANSACTION_ROLLEDBACK:   vmcid: 0x2000  minor code: 1806 completed: Maybe2022-09-15 14:41:10Caused by: org.omg.CORBA.TRANSACTION_ROLLEDBACK: ----------BEGIN server-side stack trace----------2022-09-15 14:41:10	... 103 common frames omitted2022-09-15 14:41:10	at de.symeda.sormas.api.contact.__ContactFacade_Remote_DynamicStub.getContactCountsByCasesForDashboard(de/symeda/sormas/api/contact/__ContactFacade_Remote_DynamicStub.java)2022-09-15 14:41:10	at com.sun.corba.ee.impl.presentation.rmi.codegen.CodegenStubBase.invoke(CodegenStubBase.java:226)2022-09-15 14:41:10	at com.sun.corba.ee.impl.presentation.rmi.StubInvocationHandlerImpl.invoke(StubInvocationHandlerImpl.java:140)2022-09-15 14:41:10	at com.sun.corba.ee.impl.presentation.rmi.StubInvocationHandlerImpl.privateInvoke(StubInvocationHandlerImpl.java:201)2022-09-15 14:41:10	at com.sun.corba.ee.impl.javax.rmi.CORBA.Util.mapSystemException(Util.java:277)2022-09-15 14:41:10----------END server-side stack trace----------  vmcid: 0x2000  minor code: 1806 completed: Maybe2022-09-15 14:41:102022-09-15 14:41:10	... 120 more2022-09-15 14:41:10	at com.sun.ejb.containers.EJBObjectInvocationHandler.invoke(EJBObjectInvocationHandler.java:205)2022-09-15 14:41:10	at com.sun.ejb.containers.BaseContainer.intercept(BaseContainer.java:4795)2022-09-15 14:41:10	at com.sun.ejb.containers.BaseContainer.__intercept(BaseContainer.java:4807)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.InterceptorManager.intercept(InterceptorManager.java:375)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)2022-09-15 14:41:10	at java.base/java.lang.reflect.Method.invoke(Method.java:566)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)2022-09-15 14:41:10	at jdk.internal.reflect.GeneratedMethodAccessor241.invoke(Unknown Source)2022-09-15 14:41:10	at org.jboss.weld.module.ejb.SessionBeanInterceptor.aroundInvoke(SessionBeanInterceptor.java:52)2022-09-15 14:41:10	at org.jboss.weld.module.ejb.AbstractEJBRequestScopeActivationInterceptor.aroundInvoke(AbstractEJBRequestScopeActivationInterceptor.java:81)2022-09-15 14:41:10	at com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)2022-09-15 14:41:10	at java.base/java.lang.reflect.Method.invoke(Method.java:566)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)2022-09-15 14:41:10	at jdk.internal.reflect.GeneratedMethodAccessor251.invoke(Unknown Source)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.SystemInterceptorProxy.aroundInvoke(SystemInterceptorProxy.java:140)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.SystemInterceptorProxy.doCall(SystemInterceptorProxy.java:163)2022-09-15 14:41:10	at com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)2022-09-15 14:41:10	at java.base/java.lang.reflect.Method.invoke(Method.java:566)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)2022-09-15 14:41:10	at jdk.internal.reflect.GeneratedMethodAccessor252.invoke(Unknown Source)2022-09-15 14:41:10	at de.symeda.sormas.backend.audit.AuditLoggerInterceptor.logAudit(AuditLoggerInterceptor.java:78)2022-09-15 14:41:10	at com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)2022-09-15 14:41:10	at java.base/java.lang.reflect.Method.invoke(Method.java:566)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)2022-09-15 14:41:10	at jdk.internal.reflect.GeneratedMethodAccessor253.invoke(Unknown Source)2022-09-15 14:41:10	at de.symeda.sormas.backend.util.PerformanceLoggingInterceptor.logInvokeDuration(PerformanceLoggingInterceptor.java:37)2022-09-15 14:41:10	at com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)2022-09-15 14:41:10	at java.base/java.lang.reflect.Method.invoke(Method.java:566)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)2022-09-15 14:41:10	at jdk.internal.reflect.GeneratedMethodAccessor254.invoke(Unknown Source)2022-09-15 14:41:10	at de.symeda.sormas.backend.util.RightsAllowedInterceptor.checkRightsAllowed(RightsAllowedInterceptor.java:111)2022-09-15 14:41:10	at com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)2022-09-15 14:41:10	at java.base/java.lang.reflect.Method.invoke(Method.java:566)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)2022-09-15 14:41:10	at jdk.internal.reflect.GeneratedMethodAccessor255.invoke(Unknown Source)2022-09-15 14:41:10	at de.symeda.sormas.backend.util.ValidationConstraintViolationInterceptor.handleValidationConstraintViolation(ValidationConstraintViolationInterceptor.java:35)2022-09-15 14:41:10	at com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:834)2022-09-15 14:41:10	at com.sun.ejb.EjbInvocation.invokeBeanMethod(EjbInvocation.java:665)2022-09-15 14:41:10	at com.sun.ejb.containers.BaseContainer.invokeBeanMethod(BaseContainer.java:4835)2022-09-15 14:41:10	at org.glassfish.ejb.security.application.EJBSecurityManager.invoke(EJBSecurityManager.java:408)2022-09-15 14:41:10	at org.glassfish.ejb.security.application.EJBSecurityManager.runMethod(EJBSecurityManager.java:588)2022-09-15 14:41:10	at java.base/java.lang.reflect.Method.invoke(Method.java:566)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)2022-09-15 14:41:10	at de.symeda.sormas.backend.contact.ContactFacadeEjb.getContactCountsByCasesForDashboard(ContactFacadeEjb.java:1295)2022-09-15 14:41:10	at com.sun.enterprise.container.common.impl.EntityManagerWrapper.createQuery(EntityManagerWrapper.java:498)2022-09-15 14:41:10	at org.hibernate.internal.AbstractSessionImpl.createQuery(AbstractSessionImpl.java:23)2022-09-15 14:41:10	at org.hibernate.internal.AbstractSharedSessionContract.createQuery(AbstractSharedSessionContract.java:774)2022-09-15 14:41:10	at org.hibernate.query.criteria.internal.compile.CriteriaCompiler.compile(CriteriaCompiler.java:165)2022-09-15 14:41:10	at org.hibernate.query.criteria.internal.CriteriaQueryImpl$1.buildCompiledQuery(CriteriaQueryImpl.java:364)2022-09-15 14:41:10	at org.hibernate.query.criteria.internal.compile.CriteriaCompiler$1$1.bind(CriteriaCompiler.java:135)2022-09-15 14:41:10	at org.hibernate.query.internal.AbstractProducedQuery.setParameter(AbstractProducedQuery.java:122)2022-09-15 14:41:10	at org.hibernate.query.internal.AbstractProducedQuery.setParameter(AbstractProducedQuery.java:501)2022-09-15 14:41:10	at org.hibernate.query.internal.QueryParameterBindingImpl.setBindValue(QueryParameterBindingImpl.java:55)2022-09-15 14:41:10	at org.hibernate.query.internal.QueryParameterBindingImpl.validate(QueryParameterBindingImpl.java:90)2022-09-15 14:41:10	at org.hibernate.query.spi.QueryParameterBindingValidator.validate(QueryParameterBindingValidator.java:27)2022-09-15 14:41:10	at org.hibernate.query.spi.QueryParameterBindingValidator.validate(QueryParameterBindingValidator.java:54)2022-09-15 14:41:10Caused by: java.lang.IllegalArgumentException: Parameter value [8896888] did not match expected type [de.symeda.sormas.backend.caze.Case (n/a)]2022-09-15 14:41:10	... 122 more2022-09-15 14:41:10	at com.sun.ejb.containers.BaseContainer.postInvoke(BaseContainer.java:2134)2022-09-15 14:41:10	at com.sun.ejb.containers.BaseContainer.postInvokeTx(BaseContainer.java:4601)2022-09-15 14:41:10	at com.sun.ejb.containers.EJBContainerTransactionManager.postInvokeTx(EJBContainerTransactionManager.java:485)2022-09-15 14:41:10	at com.sun.ejb.containers.EJBContainerTransactionManager.checkExceptionClientTx(EJBContainerTransactionManager.java:615)2022-09-15 14:41:10Caused by: javax.ejb.TransactionRolledbackLocalException: Exception thrown from bean: java.lang.IllegalArgumentException: Parameter value [8896888] did not match expected type [de.symeda.sormas.backend.caze.Case (n/a)]2022-09-15 14:41:10	at java.base/java.lang.Thread.run(Thread.java:829)2022-09-15 14:41:10	at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.run(AbstractThreadPool.java:549)2022-09-15 14:41:10	at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.doWork(AbstractThreadPool.java:569)2022-09-15 14:41:10	at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy$WorkerThreadRunnable.run(WorkerThreadIOStrategy.java:114)2022-09-15 14:41:10	at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.access$100(WorkerThreadIOStrategy.java:33)2022-09-15 14:41:10	at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.run0(WorkerThreadIOStrategy.java:94)2022-09-15 14:41:10	at org.glassfish.grizzly.strategies.AbstractIOStrategy.fireIOEvent(AbstractIOStrategy.java:89)2022-09-15 14:41:10	at org.glassfish.grizzly.nio.transport.TCPNIOTransport.fireIOEvent(TCPNIOTransport.java:524)2022-09-15 14:41:10	at org.glassfish.grizzly.ProcessorExecutor.execute(ProcessorExecutor.java:53)2022-09-15 14:41:10	at org.glassfish.grizzly.filterchain.DefaultFilterChain.process(DefaultFilterChain.java:88)2022-09-15 14:41:10	at org.glassfish.grizzly.filterchain.DefaultFilterChain.execute(DefaultFilterChain.java:109)2022-09-15 14:41:10	at org.glassfish.grizzly.filterchain.DefaultFilterChain.executeChainPart(DefaultFilterChain.java:177)2022-09-15 14:41:10	at org.glassfish.grizzly.filterchain.DefaultFilterChain.executeFilter(DefaultFilterChain.java:260)2022-09-15 14:41:10	at org.glassfish.grizzly.filterchain.ExecutorResolver$9.execute(ExecutorResolver.java:95)2022-09-15 14:41:10	at org.glassfish.grizzly.http.server.HttpServerFilter.handleRead(HttpServerFilter.java:218)2022-09-15 14:41:10	at org.glassfish.grizzly.http.server.HttpHandler.doHandle(HttpHandler.java:156)2022-09-15 14:41:10	at org.glassfish.grizzly.http.server.HttpHandler.runService(HttpHandler.java:182)2022-09-15 14:41:10	at com.sun.enterprise.v3.services.impl.ContainerMapper.service(ContainerMapper.java:217)2022-09-15 14:41:10	at com.sun.enterprise.v3.services.impl.ContainerMapper$HttpHandlerCallable.call(ContainerMapper.java:520)2022-09-15 14:41:10	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:238)2022-09-15 14:41:10	at org.apache.catalina.connector.CoyoteAdapter.doService(CoyoteAdapter.java:371)2022-09-15 14:41:10	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:158)2022-09-15 14:41:10	at com.sun.enterprise.web.WebPipeline.invoke(WebPipeline.java:97)2022-09-15 14:41:10	at org.apache.catalina.core.StandardPipeline.doChainInvoke(StandardPipeline.java:581)2022-09-15 14:41:10	at org.apache.catalina.core.StandardPipeline.doInvoke(StandardPipeline.java:757)2022-09-15 14:41:10	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:160)2022-09-15 14:41:10	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:257)2022-09-15 14:41:10	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:211)2022-09-15 14:41:10	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:253)2022-09-15 14:41:10	at de.symeda.sormas.ui.Slf4jMdcFilter.doFilter(Slf4jMdcFilter.java:29)2022-09-15 14:41:10	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:211)2022-09-15 14:41:10	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:253)2022-09-15 14:41:10	at de.symeda.sormas.ui.SessionFilter.doFilter(SessionFilter.java:66)2022-09-15 14:41:10	at de.symeda.sormas.ui.__EJB31_Generated__SessionFilterBean__Intf____Bean__.doFilter(Unknown Source)2022-09-15 14:41:10	at com.sun.proxy.$Proxy893.doFilter(Unknown Source)2022-09-15 14:41:10	at com.sun.ejb.containers.EJBLocalObjectInvocationHandlerDelegate.invoke(EJBLocalObjectInvocationHandlerDelegate.java:90)2022-09-15 14:41:10	at com.sun.ejb.containers.EJBLocalObjectInvocationHandler.invoke(EJBLocalObjectInvocationHandler.java:212)2022-09-15 14:41:10	at com.sun.ejb.containers.BaseContainer.intercept(BaseContainer.java:4795)2022-09-15 14:41:10	at com.sun.ejb.containers.BaseContainer.__intercept(BaseContainer.java:4807)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.InterceptorManager.intercept(InterceptorManager.java:375)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)2022-09-15 14:41:10	at java.base/java.lang.reflect.Method.invoke(Method.java:566)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)2022-09-15 14:41:10	at jdk.internal.reflect.GeneratedMethodAccessor241.invoke(Unknown Source)2022-09-15 14:41:10	at org.jboss.weld.module.ejb.SessionBeanInterceptor.aroundInvoke(SessionBeanInterceptor.java:52)2022-09-15 14:41:10	at org.jboss.weld.module.ejb.AbstractEJBRequestScopeActivationInterceptor.aroundInvoke(AbstractEJBRequestScopeActivationInterceptor.java:72)2022-09-15 14:41:10	at com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)2022-09-15 14:41:10	at java.base/java.lang.reflect.Method.invoke(Method.java:566)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)2022-09-15 14:41:10	at jdk.internal.reflect.GeneratedMethodAccessor251.invoke(Unknown Source)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.SystemInterceptorProxy.aroundInvoke(SystemInterceptorProxy.java:140)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.SystemInterceptorProxy.doCall(SystemInterceptorProxy.java:163)2022-09-15 14:41:10	at com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)2022-09-15 14:41:10	at com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:834)2022-09-15 14:41:10	at com.sun.ejb.EjbInvocation.invokeBeanMethod(EjbInvocation.java:665)2022-09-15 14:41:10	at com.sun.ejb.containers.BaseContainer.invokeBeanMethod(BaseContainer.java:4835)2022-09-15 14:41:10	at org.glassfish.ejb.security.application.EJBSecurityManager.invoke(EJBSecurityManager.java:408)2022-09-15 14:41:10	at org.glassfish.ejb.security.application.EJBSecurityManager.runMethod(EJBSecurityManager.java:588)2022-09-15 14:41:10	at java.base/java.lang.reflect.Method.invoke(Method.java:566)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)2022-09-15 14:41:10	at jdk.internal.reflect.GeneratedMethodAccessor467.invoke(Unknown Source)2022-09-15 14:41:10	at de.symeda.sormas.ui.SessionFilterBean.doFilter(SessionFilterBean.java:37)2022-09-15 14:41:10	at de.symeda.sormas.ui.SessionFilter.lambda$doFilter$2(SessionFilter.java:73)2022-09-15 14:41:10	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:211)2022-09-15 14:41:10	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:331)2022-09-15 14:41:10	at org.apache.catalina.core.StandardWrapper.service(StandardWrapper.java:1637)2022-09-15 14:41:10	at javax.servlet.http.HttpServlet.service(HttpServlet.java:750)2022-09-15 14:41:10	at com.vaadin.server.VaadinServlet.service(VaadinServlet.java:464)2022-09-15 14:41:10	at com.vaadin.server.VaadinService.handleRequest(VaadinService.java:1637)2022-09-15 14:41:10	at com.vaadin.server.SynchronizedRequestHandler.handleRequest(SynchronizedRequestHandler.java:40)2022-09-15 14:41:10	at com.vaadin.server.communication.UidlRequestHandler.synchronizedHandleRequest(UidlRequestHandler.java:91)2022-09-15 14:41:10	at com.vaadin.server.communication.ServerRpcHandler.handleRpc(ServerRpcHandler.java:275)2022-09-15 14:41:10	at com.vaadin.server.communication.ServerRpcHandler.handleInvocations(ServerRpcHandler.java:407)2022-09-15 14:41:10	at com.vaadin.server.communication.ServerRpcHandler.handleInvocation(ServerRpcHandler.java:442)2022-09-15 14:41:10	at com.vaadin.server.ServerRpcManager.applyInvocation(ServerRpcManager.java:116)2022-09-15 14:41:10	at com.vaadin.server.ServerRpcManager.applyInvocation(ServerRpcManager.java:155)2022-09-15 14:41:10	at java.base/java.lang.reflect.Method.invoke(Method.java:566)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)2022-09-15 14:41:10	at jdk.internal.reflect.GeneratedMethodAccessor787.invoke(Unknown Source)2022-09-15 14:41:10	at com.vaadin.ui.Button$1.click(Button.java:57)2022-09-15 14:41:10	at com.vaadin.ui.Button.fireClick(Button.java:384)2022-09-15 14:41:10	at com.vaadin.server.AbstractClientConnector.fireEvent(AbstractClientConnector.java:1190)2022-09-15 14:41:10	at com.vaadin.event.EventRouter.fireEvent(EventRouter.java:363)2022-09-15 14:41:10	at com.vaadin.event.EventRouter.fireEvent(EventRouter.java:399)2022-09-15 14:41:10	at com.vaadin.event.ListenerMethod.receiveEvent(ListenerMethod.java:706)2022-09-15 14:41:10	at java.base/java.lang.reflect.Method.invoke(Method.java:566)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)2022-09-15 14:41:10	at jdk.internal.reflect.GeneratedMethodAccessor769.invoke(Unknown Source)2022-09-15 14:41:10	at de.symeda.sormas.ui.Menu.lambda$createViewButton$f7d2215a$1(Menu.java:196)2022-09-15 14:41:10	at com.vaadin.navigator.Navigator.navigateTo(Navigator.java:678)2022-09-15 14:41:10	at com.vaadin.navigator.Navigator.navigateTo(Navigator.java:701)2022-09-15 14:41:10	at com.vaadin.navigator.Navigator.runAfterLeaveConfirmation(Navigator.java:730)2022-09-15 14:41:10	at com.vaadin.navigator.View.beforeLeave(View.java:79)2022-09-15 14:41:10	at com.vaadin.navigator.ViewBeforeLeaveEvent.navigate(ViewBeforeLeaveEvent.java:54)2022-09-15 14:41:10	at com.vaadin.navigator.Navigator.lambda$navigateTo$9a874efd$1(Navigator.java:702)2022-09-15 14:41:10	at com.vaadin.navigator.Navigator.performNavigateTo(Navigator.java:778)2022-09-15 14:41:10	at de.symeda.sormas.ui.dashboard.contacts.ContactsDashboardView.enter(ContactsDashboardView.java:144)2022-09-15 14:41:10	at de.symeda.sormas.ui.dashboard.contacts.ContactsDashboardView.refreshDashboard(ContactsDashboardView.java:475)2022-09-15 14:41:10	at de.symeda.sormas.ui.dashboard.contacts.ContactsDashboardView.updateCaseCountsAndSourceCasesLabels(ContactsDashboardView.java:269)2022-09-15 14:41:10	at de.symeda.sormas.api.contact._ContactFacade_Wrapper.getContactCountsByCasesForDashboard(de/symeda/sormas/api/contact/_ContactFacade_Wrapper.java)2022-09-15 14:41:10	at de.symeda.sormas.api.contact.__ContactFacade_Remote_DynamicStub.getContactCountsByCasesForDashboard(de/symeda/sormas/api/contact/__ContactFacade_Remote_DynamicStub.java)2022-09-15 14:41:10	at com.sun.corba.ee.impl.presentation.rmi.codegen.CodegenStubBase.invoke(CodegenStubBase.java:226)2022-09-15 14:41:10	at com.sun.corba.ee.impl.presentation.rmi.StubInvocationHandlerImpl.invoke(StubInvocationHandlerImpl.java:140)2022-09-15 14:41:10	at com.sun.corba.ee.impl.presentation.rmi.StubInvocationHandlerImpl.privateInvoke(StubInvocationHandlerImpl.java:188)2022-09-15 14:41:10	at com.sun.corba.ee.impl.protocol.ClientDelegateImpl.invoke(ClientDelegateImpl.java:258)2022-09-15 14:41:10	at com.sun.corba.ee.impl.protocol.SharedCDRClientRequestDispatcherImpl.marshalingComplete(SharedCDRClientRequestDispatcherImpl.java:128)2022-09-15 14:41:10	at com.sun.corba.ee.impl.protocol.MessageMediatorImpl.handleRequestRequest(MessageMediatorImpl.java:1565)2022-09-15 14:41:10	at com.sun.corba.ee.impl.protocol.ServerRequestDispatcherImpl.dispatch(ServerRequestDispatcherImpl.java:199)2022-09-15 14:41:10	at com.sun.corba.ee.impl.protocol.ServerRequestDispatcherImpl.dispatchToServant(ServerRequestDispatcherImpl.java:528)2022-09-15 14:41:10	at com.sun.corba.ee.impl.presentation.rmi.ReflectiveTie._invoke(ReflectiveTie.java:173)2022-09-15 14:41:10	at com.sun.corba.ee.impl.presentation.rmi.ReflectiveTie.dispatchToMethod(ReflectiveTie.java:143)2022-09-15 14:41:10	at java.base/java.lang.reflect.Method.invoke(Method.java:566)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)2022-09-15 14:41:10	at com.sun.proxy.$Proxy682.getContactCountsByCasesForDashboard(Unknown Source)2022-09-15 14:41:10	at com.sun.ejb.containers.EJBObjectInvocationHandlerDelegate.invoke(EJBObjectInvocationHandlerDelegate.java:79)2022-09-15 14:41:10	at com.sun.ejb.containers.EJBObjectInvocationHandler.invoke(EJBObjectInvocationHandler.java:212)2022-09-15 14:41:10	at com.sun.ejb.containers.BaseContainer.postInvoke(BaseContainer.java:2104)2022-09-15 14:41:10	at com.sun.ejb.containers.BaseContainer.postInvoke(BaseContainer.java:2173)2022-09-15 14:41:10	at com.sun.ejb.containers.BaseContainer.mapRemoteException(BaseContainer.java:2330)2022-09-15 14:41:10	at com.sun.ejb.containers.BaseContainer.mapRemoteException(BaseContainer.java:2357)2022-09-15 14:41:10	at org.glassfish.enterprise.iiop.impl.POAProtocolMgr.mapException(POAProtocolMgr.java:356)2022-09-15 14:41:10org.omg.CORBA.TRANSACTION_ROLLEDBACK:   vmcid: 0x2000  minor code: 1806 completed: Maybe2022-09-15 14:41:10	org.omg.CORBA.TRANSACTION_ROLLEDBACK: ----------BEGIN server-side stack trace----------2022-09-15 14:41:10Caused by: javax.transaction.TransactionRolledbackException: CORBA TRANSACTION_ROLLEDBACK 9998 Maybe; nested exception is: 2022-09-15 14:41:10	... 87 common frames omitted2022-09-15 14:41:10	at com.vaadin.event.ListenerMethod.receiveEvent(ListenerMethod.java:706)2022-09-15 14:41:10	at java.base/java.lang.reflect.Method.invoke(Method.java:566)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)2022-09-15 14:41:10	at jdk.internal.reflect.GeneratedMethodAccessor769.invoke(Unknown Source)2022-09-15 14:41:10	at de.symeda.sormas.ui.Menu.lambda$createViewButton$f7d2215a$1(Menu.java:196)2022-09-15 14:41:10	at com.vaadin.navigator.Navigator.navigateTo(Navigator.java:678)2022-09-15 14:41:10	at com.vaadin.navigator.Navigator.navigateTo(Navigator.java:701)2022-09-15 14:41:10	at com.vaadin.navigator.Navigator.runAfterLeaveConfirmation(Navigator.java:730)2022-09-15 14:41:10	at com.vaadin.navigator.View.beforeLeave(View.java:79)2022-09-15 14:41:10	at com.vaadin.navigator.ViewBeforeLeaveEvent.navigate(ViewBeforeLeaveEvent.java:54)2022-09-15 14:41:10	at com.vaadin.navigator.Navigator.lambda$navigateTo$9a874efd$1(Navigator.java:702)2022-09-15 14:41:10	at com.vaadin.navigator.Navigator.performNavigateTo(Navigator.java:778)2022-09-15 14:41:10	at de.symeda.sormas.ui.dashboard.contacts.ContactsDashboardView.enter(ContactsDashboardView.java:144)2022-09-15 14:41:10	at de.symeda.sormas.ui.dashboard.contacts.ContactsDashboardView.refreshDashboard(ContactsDashboardView.java:475)2022-09-15 14:41:10	at de.symeda.sormas.ui.dashboard.contacts.ContactsDashboardView.updateCaseCountsAndSourceCasesLabels(ContactsDashboardView.java:269)2022-09-15 14:41:10	at de.symeda.sormas.api.contact._ContactFacade_Wrapper.getContactCountsByCasesForDashboard(de/symeda/sormas/api/contact/_ContactFacade_Wrapper.java)2022-09-15 14:41:10Caused by: javax.ejb.EJBTransactionRolledbackException: null2022-09-15 14:41:10	... 82 common frames omitted2022-09-15 14:41:10	at com.vaadin.ui.Button$1.click(Button.java:57)2022-09-15 14:41:10	at com.vaadin.ui.Button.fireClick(Button.java:384)2022-09-15 14:41:10	at com.vaadin.server.AbstractClientConnector.fireEvent(AbstractClientConnector.java:1190)2022-09-15 14:41:10	at com.vaadin.event.EventRouter.fireEvent(EventRouter.java:363)2022-09-15 14:41:10	at com.vaadin.event.EventRouter.fireEvent(EventRouter.java:399)2022-09-15 14:41:10	at com.vaadin.event.ListenerMethod.receiveEvent(ListenerMethod.java:726)2022-09-15 14:41:10Caused by: com.vaadin.event.ListenerMethod$MethodException: Invocation of method buttonClick in de.symeda.sormas.ui.Menu$$Lambda$2457/0x00000008021b6c40 failed.2022-09-15 14:41:10	... 78 common frames omitted2022-09-15 14:41:10	at com.vaadin.server.ServerRpcManager.applyInvocation(ServerRpcManager.java:155)2022-09-15 14:41:10	at java.base/java.lang.reflect.Method.invoke(Method.java:566)2022-09-15 14:41:10	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)2022-09-15 14:41:10	at jdk.internal.reflect.GeneratedMethodAccessor787.invoke(Unknown Source)2022-09-15 14:41:10Caused by: java.lang.reflect.InvocationTargetException: null2022-09-15 14:41:10	at java.base/java.lang.Thread.run(Thread.java:829)2022-09-15 14:41:10	at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.run(AbstractThreadPool.java:549)2022-09-15 14:41:10	at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.doWork(AbstractThreadPool.java:569)2022-09-15 14:41:10	at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy$WorkerThreadRunnable.run(WorkerThreadIOStrategy.java:114)2022-09-15 14:41:10	at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.access$100(WorkerThreadIOStrategy.java:33)2022-09-15 14:41:10	at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.run0(WorkerThreadIOStrategy.java:94)2022-09-15 14:41:10	at org.glassfish.grizzly.strategies.AbstractIOStrategy.fireIOEvent(AbstractIOStrategy.java:89)2022-09-15 14:41:10	at org.glassfish.grizzly.nio.transport.TCPNIOTransport.fireIOEvent(TCPNIOTransport.java:524)2022-09-15 14:41:10	at org.glassfish.grizzly.ProcessorExecutor.execute(ProcessorExecutor.java:53)2022-09-15 14:41:10	at org.glassfish.grizzly.filterchain.DefaultFilterChain.process(DefaultFilterChain.java:88)2022-09-15 14:41:10	at org.glassfish.grizzly.filterchain.DefaultFilterChain.execute(DefaultFilterChain.java:109)2022-09-15 14:41:10	at org.glassfish.grizzly.filterchain.DefaultFilterChain.executeChainPart(DefaultFilterChain.java:177)2022-09-15 14:41:10	at org.glassfish.grizzly.filterchain.DefaultFilterChain.executeFilter(DefaultFilterChain.java:260)2022-09-15 14:41:10	at org.glassfish.grizzly.filterchain.ExecutorResolver$9.execute(ExecutorResolver.java:95)2022-09-15 14:41:10	at org.glassfish.grizzly.http.server.HttpServerFilter.handleRead(HttpServerFilter.java:218)2022-09-15 14:41:10	at org.glassfish.grizzly.http.server.HttpHandler.doHandle(HttpHandler.java:156)2022-09-15 14:41:10	at org.glassfish.grizzly.http.server.HttpHandler.runService(HttpHandler.java:182)2022-09-15 14:41:10	at com.sun.enterprise.v3.services.impl.ContainerMapper.service(ContainerMapper.java:217)2022-09-15 14:41:10	at com.sun.enterprise.v3.services.impl.ContainerMapper$HttpHandlerCallable.call(ContainerMapper.java:520)2022-09-15 14:41:10	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:238)2022-09-15 14:41:10	at org.apache.catalina.connector.CoyoteAdapter.doService(CoyoteAdapter.java:371)2022-09-15 14:41:10	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:158)2022-09-15 14:41:10	at com.sun.enterprise.web.WebPipeline.invoke(WebPipeline.java:97)2022-09-15 14:41:10	at org.apache.catalina.core.StandardPipeline.doChainInvoke(StandardPipeline.java:581)2022-09-15 14:41:10	at org.apache.catalina.core.StandardPipeline.doInvoke(StandardPipeline.java:757)2022-09-15 14:41:10	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:160)2022-09-15 14:41:10	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:257)2022-09-15 14:41:10	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:211)2022-09-15 14:41:10	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:253)2022-09-15 14:41:10	at de.symeda.sormas.ui.Slf4jMdcFilter.doFilter(Slf4jMdcFilter.java:29)2022-09-15 14:41:10	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:211)

--- a/sormas-ui/src/main/java/de/symeda/sormas/ui/dashboard/AbstractDashboardView.java
+++ b/sormas-ui/src/main/java/de/symeda/sormas/ui/dashboard/AbstractDashboardView.java
@@ -19,6 +19,7 @@ package de.symeda.sormas.ui.dashboard;
 
 import static de.symeda.sormas.ui.UiUtil.permitted;
 
+import com.vaadin.navigator.ViewChangeListener.ViewChangeEvent;
 import com.vaadin.ui.VerticalLayout;
 import com.vaadin.ui.themes.ValoTheme;
 import com.vaadin.v7.data.Property;
@@ -88,6 +89,11 @@ public abstract class AbstractDashboardView extends AbstractView {
 		} else {
 			SormasUI.get().getNavigator().navigateTo(CampaignDashboardView.VIEW_NAME);
 		}
+	}
+
+	@Override
+	public void enter(ViewChangeEvent event) {
+		refreshDashboard();
 	}
 
 	public abstract void refreshDashboard();

--- a/sormas-ui/src/main/java/de/symeda/sormas/ui/dashboard/DashboardDataProvider.java
+++ b/sormas-ui/src/main/java/de/symeda/sormas/ui/dashboard/DashboardDataProvider.java
@@ -177,7 +177,7 @@ public class DashboardDataProvider {
 		setCaseWithReferenceDefinitionFulfilledCount(Long.valueOf(casesWithReferenceDefinitionFulfilled.size()));
 	}
 
-	private void refreshDataForSelectedDisease() {
+	public void refreshDataForSelectedDisease() {
 
 		// Update the entities lists according to the filters
 
@@ -358,8 +358,6 @@ public class DashboardDataProvider {
 
 	public void setDisease(Disease disease) {
 		this.disease = disease;
-
-		this.refreshDataForSelectedDisease();
 	}
 
 	public Date getFromDate() {

--- a/sormas-ui/src/main/java/de/symeda/sormas/ui/dashboard/campaigns/CampaignDashboardView.java
+++ b/sormas-ui/src/main/java/de/symeda/sormas/ui/dashboard/campaigns/CampaignDashboardView.java
@@ -9,7 +9,6 @@ import java.util.stream.Collectors;
 
 import org.apache.commons.lang3.StringUtils;
 
-import com.vaadin.navigator.ViewChangeListener;
 import com.vaadin.server.Page;
 import com.vaadin.shared.ui.MarginInfo;
 import com.vaadin.ui.Component;
@@ -60,11 +59,6 @@ public class CampaignDashboardView extends AbstractDashboardView {
 		filterLayout.setInfoLabelText(I18nProperties.getString(Strings.infoCampaignsDashboard));
 		dashboardLayout.setExpandRatio(filterLayout, 0);
 		dashboardLayout.setSizeFull();
-	}
-
-	@Override
-	public void enter(ViewChangeListener.ViewChangeEvent event) {
-		refreshDashboard();
 	}
 
 	@SuppressWarnings("deprecation")

--- a/sormas-ui/src/main/java/de/symeda/sormas/ui/dashboard/contacts/ContactsDashboardView.java
+++ b/sormas-ui/src/main/java/de/symeda/sormas/ui/dashboard/contacts/ContactsDashboardView.java
@@ -22,7 +22,6 @@ import java.util.Optional;
 import java.util.stream.Collectors;
 
 import com.vaadin.icons.VaadinIcons;
-import com.vaadin.navigator.ViewChangeListener;
 import com.vaadin.shared.ui.ContentMode;
 import com.vaadin.shared.ui.MarginInfo;
 import com.vaadin.ui.Alignment;
@@ -137,11 +136,6 @@ public class ContactsDashboardView extends AbstractDashboardView {
 			networkDiagramRowLayout = createNetworkDiagramRowLayout();
 			rowsLayout.addComponent(networkDiagramRowLayout);
 		}
-	}
-
-	@Override
-	public void enter(ViewChangeListener.ViewChangeEvent event) {
-		refreshDashboard();
 	}
 
 	private HorizontalLayout createCaseStatisticsLayout() {

--- a/sormas-ui/src/main/java/de/symeda/sormas/ui/dashboard/contacts/components/ContactsFilterLayout.java
+++ b/sormas-ui/src/main/java/de/symeda/sormas/ui/dashboard/contacts/components/ContactsFilterLayout.java
@@ -59,6 +59,7 @@ public class ContactsFilterLayout extends DashboardFilterLayout {
 		diseaseFilter.setValue(dashboardDataProvider.getDisease());
 		diseaseFilter.addValueChangeListener(e -> {
 			dashboardDataProvider.setDisease((Disease) diseaseFilter.getValue());
+			dashboardDataProvider.refreshDataForSelectedDisease();
 		});
 		addCustomComponent(diseaseFilter, DISEASE_FILTER);
 	}

--- a/sormas-ui/src/main/java/de/symeda/sormas/ui/dashboard/surveillance/SurveillanceDashboardView.java
+++ b/sormas-ui/src/main/java/de/symeda/sormas/ui/dashboard/surveillance/SurveillanceDashboardView.java
@@ -17,8 +17,6 @@
  *******************************************************************************/
 package de.symeda.sormas.ui.dashboard.surveillance;
 
-import com.vaadin.navigator.ViewChangeListener;
-
 import de.symeda.sormas.api.FacadeProvider;
 import de.symeda.sormas.api.caze.NewCaseDateType;
 import de.symeda.sormas.ui.dashboard.AbstractDashboardView;
@@ -80,11 +78,6 @@ public class SurveillanceDashboardView extends AbstractDashboardView {
 				dashboardLayout.addComponent(surveillanceOverviewLayout, 1);
 			}
 		});
-	}
-
-	@Override
-	public void enter(ViewChangeListener.ViewChangeEvent event) {
-		refreshDashboard();
 	}
 
 	public void refreshDashboard() {

--- a/sormas-ui/src/main/java/de/symeda/sormas/ui/dashboard/surveillance/SurveillanceDiseaseCarouselLayout.java
+++ b/sormas-ui/src/main/java/de/symeda/sormas/ui/dashboard/surveillance/SurveillanceDiseaseCarouselLayout.java
@@ -186,6 +186,7 @@ public class SurveillanceDiseaseCarouselLayout extends VerticalLayout {
 	private void changeSelectedDisease(Disease disease, boolean doRefresh) {
 		this.dashboardDataProvider.setDisease(disease);
 		if (doRefresh) {
+			this.dashboardDataProvider.refreshDataForSelectedDisease();
 			refresh();
 		}
 	}

